File: 344. Reverse String.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        hl, l = len(s)//2, len(s)
        for i in range(hl):
            s[i], s[l-1-i] = s[l-1-i], s[i]

File: 344. Reverse String.java
class Solution {
    public void reverseString(char[] s) {
        int lo = 0, hi = s.length-1;
        while (hi > lo) {
            char c = s[lo];
            s[lo] = s[hi];
            s[hi] = c;
            lo++; hi--;
        }
    }
}